python/cugraph/cugraph/tests/conftest.py

```diff
@@ -23,6 +23,11 @@
 # Avoid timeout during shutdown
 from dask_cuda.utils_test import IncreasedCloseTimeoutNanny
 
+import warnings
+
+warnings.filterwarnings("ignore", category=DeprecationWarning, module="cugraph")
```

Comment:
I remember we discussed this offline but now I realize the right way is probably to edit `pytest.ini`, so this can also be removed.

Reply:
I need to remove this since it did not work